src/modules/position_estimator_inav/CMakeLists.txt

```diff
@@ -31,16 +31,11 @@
 #
 ############################################################################
 
-set(MODULE_CFLAGS )
-if(${OS} STREQUAL "nuttx")
-	list(APPEND MODULE_CFLAGS -Wframe-larger-than=4000)
```

Comment:
Right. I get that, but for instance inav was reporting 4k for shell
handler, but the file had 1200 listed, what should I do there. With my auto
warning on frame size that was causing inav compile to error.
On Mar 23, 2016 3:47 PM, "Lorenz Meier" <notifications@github.com> wrote:

> In src/modules/position_estimator_inav/CMakeLists.txt
> <https://github.com/PX4/Firmware/pull/4036#discussion_r57234014>:
>
> > @@ -31,16 +31,11 @@
> >  #
> >  ############################################################################
> >
> > -set(MODULE_CFLAGS )
> > -if(${OS} STREQUAL "nuttx")
> > -	list(APPEND MODULE_CFLAGS -Wframe-larger-than=4000)
>
> There is one catch: The stack size in the CMake file is NOT the stack of
> the daemon. Its the stack of the shell handler. The relationship is that
> the stack size provided to pthread_create() and px4_task_spawn needs to be
> larger than the frame size. The CMake stack size is only relevant for
> non-daemons, e.g. the pwm app.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/PX4/Firmware/pull/4036/files/88208d0907471173f9234ebeab3171ca7b187c8b#r57234014>
>